h/browser/chrome/lib/sidebar-injector.js

```diff
@@ -3,6 +3,32 @@
 var blocklist = require('../../../static/scripts/blocklist');
 var errors = require('./errors');
 var settings = require('./settings');
+var util = require('./util');
+
+var CONTENT_TYPE_HTML = 'HTML';
+var CONTENT_TYPE_PDF = 'PDF';
+
+// a function which is executed as a content script
+// to determine the type of content being displayed in a tab
```

Comment:
I agree on both counts. I originally only used JSDoc comments for the public API of a module but after doing some testing with a couple of tools which should documentation for functions at the point where you use them (Visual Studio Code, Atom), both will show the first sentence of JSDoc but not inline comments.